src/probnum/random_variables/_normal.py

```diff
@@ -413,8 +430,10 @@ def _sub_normal(self, other: "Normal") -> "Normal":
         )
 
     # Univariate Gaussians
-    def _univariate_cov_cholesky(self) -> np.floating:
-        return np.sqrt(self._cov)
+    def _univariate_cov_cholesky(
+        self, damping_factor=COV_CHOLESKY_DAMPING
```

Comment:
all of these functions have a default damping parameter now. Similarly to before, only that now it is easier to change it.